embedded-graphics/src/image/mod.rs

```diff
@@ -15,6 +15,9 @@
 //! This example loads a TGA-formatted image using the [tinytga] crate and draws it to the display
 //! using the [`Image`] wrapper. The image is positioned at the top left corner of the display.
 //!
+//! The `graphics` feature of `tinytga` needs to be enabled in `Cargo.toml` to use the `Tga` object
+//! with embedded-graphics.
```

Comment:
I was lazy when I created the last review and forgot to mention that this suggestion was meant to also apply to the other instances that describe the need for the `graphics` feature. IMO it's especially important to change this in `lib.rs`.

Reply:
No problem, I've updated the other places too. Tbf I was lazy when I copypasta'd this comment throughout the codebase.